### ISSUE TITLE
Spelling mistake in pb-copyable.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6440]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6440|
|Type|defect|
|Created|2013-04-17 13:19:35Z|

`doc/core/howto/pb-copyable.xhtml` says "eligble" rather than "eligible".

Attachments:

* [pb-copyable_xhtml.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/0c1/0c1240071a8b9be51373fae7e901df46e87c95d5/31145b787b7942d162e264505e8131be81daec05.patch) (714 bytes) - added by chitrankdixit on 2013-04-21 05:52:42Z - spelling mistake "eligble" changed to "eligible"

<details><summary>Searchable metadata</summary>

```
trac-id__6440 6440
type__defect defect
reporter__itamar itamar
priority__lowest lowest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1366204775000000 1366204775000000
changetime__1366914988000000 1366914988000000
version__None None
owner__thijs thijs

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Infinity's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Infinity set owner to Infinity<br>Infinity set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="Infinity's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Infinity set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="Infinity's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Infinity removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch, I'll apply it. Make sure to create your patch from the root of your checkout next time, as described [TwistedDevelopment#SubmittingaPatch here].

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38259]) Apply pb-copyable_xhtml.patch: Correct a typo in the PB copyable howto.

Author: chitrankdixit
Reviewer: thijs
Fixes: [#6440](https://github.com/twisted/twisted/issues/6440)
